src/System.Net.HttpListener/src/Resources/Strings.resx

```diff
@@ -298,39 +357,33 @@
   <data name="net_log_operation_failed_with_error" xml:space="preserve">
     <value>{0} failed with error {1}.</value>
   </data>
-  <data name="net_MethodNotImplementedException" xml:space="preserve">
-    <value>This method is not implemented by this class.</value>
-  </data>
-  <data name="event_OperationReturnedSomething" xml:space="preserve">
-    <value>{0} returned {1}.</value>
-  </data>
   <data name="net_invalid_enum" xml:space="preserve">
     <value>The specified value is not valid in the '{0}' enumeration.</value>
   </data>
-  <data name="net_auth_message_not_encrypted" xml:space="preserve">
-    <value>Protocol error: A received message contains a valid signature but it was not encrypted as required by the effective Protection Level.</value>
-  </data>
   <data name="SSPIInvalidHandleType" xml:space="preserve">
     <value>'{0}' is not a supported handle type.</value>
   </data>
-  <data name="offset_out_of_range" xml:space="preserve">
-    <value>Offset exceeds the length of buffer.</value>
-  </data>
   <data name="net_io_operation_aborted" xml:space="preserve">
     <value>I/O operation aborted: '{0}'.</value>
   </data>
+  <data name="event_OperationReturnedSomething" xml:space="preserve">
+    <value>{0} returned {1}.</value>
+  </data>
```

Comment:
Why were these resources reordered?  It makes it difficult to understand from the diff what was removed vs moved.